tests/performance/local/CMakeLists.txt

```diff
@@ -162,9 +162,10 @@ if(HPX_WITH_CUDA_COMPUTE)
     stream.cpp PROPERTIES CUDA_SOURCE_PROPERTY_FORMAT OBJ
   )
 endif()
-if(HPX_WITH_CUDA_CLANG)
+# TODO: Necessary?
+if(HPX_WITH_CLANG_CUDA)
   set_source_files_properties(
-    stream.cpp PROPERTIES COMPILE_FLAGS "-xcuda ${HPX_CUDA_CLANG_FLAGS}"
+    stream.cpp PROPERTIES COMPILE_FLAGS "-xcuda ${CMAKE_CUDA_FLAGS}"
```

Comment:
I think the "proper" way to treat this as a cuda file is to set its type to `CUDA`. That should work for both nvcc and clang.